src/app/gigs/chat/chat-layout-minimal.tsx

```diff
@@ -46,7 +57,30 @@ export function ChatLayoutMinimal({
   stateRef.current = state;
 
   const messageHandlers: MessageCollectionEventHandler = {
-    onMessagesAdded: (context: any, channel: any, messages: any) => {
+    // Channel will always be defined whenever this event is triggered bc it is triggered thru sendbird and
+    // only runs when a channel is successfully found between two users
+    onMessagesAdded: async (context: any, channel: any, messages: any) => {
```

Comment:
```suggestion
    onMessagesAdded: async (context, channel, messages) => {
```
The type on the line above types these params. why override as `any`?

Reply:
Ahh I was getting a linting error before, but now I just removed it and now the types are properly assigned! 👍